functional/cypress-tests/cypress/support/widgetAssertions.js

```diff
@@ -59,8 +48,8 @@ export function validateJenkinsJob() {
     .should('is.visible');
   cy.contains('p', 'master-branch')
     .should('is.visible');
-  cy.contains('p', /[0-9]{2}.[0-9]{2}.[0-9]{4}, [0-9]{2}:[0-9]{2}:[0-9]{2}/)
-    .should('is.visible');
+  cy.contains('p', /[0-9]{2}.[0-9]{2}.[0-9]{4}, [0-9]{2}:[0-9]{2}:[0-9]{2}/).
+    should('is.visible');
```

Comment:
Inconsistency with the dot at the end of line; others have it in a standard way.